portal-frontend/src/app/features/notice-of-intents/edit-submission/proposal/roso/roso-proposal.component.ts

```diff
@@ -111,8 +109,7 @@ export class RosoProposalComponent extends FilesStepComponent implements OnInit,
           followUpIds: noiSubmission.soilFollowUpIDs,
           purpose: noiSubmission.purpose,
           soilTypeRemoved: noiSubmission.soilTypeRemoved,
-          projectDurationAmount: noiSubmission.soilProjectDurationAmount?.toString() ?? null,
-          projectDurationUnit: noiSubmission.soilProjectDurationUnit,
+          projectDuration: noiSubmission.soilProjectDuration?.toString() ?? null,
```

Comment:
Same thing as above RE: chained methods and conditions; it is already a string type

Reply:
changes